client/quantum_serverless/core/job.py

```diff
@@ -96,6 +96,14 @@ def result(self, job_id: str):
         """Return results."""
         raise NotImplementedError
 
+    def primitive(self, job_id: str):
```

Comment:
Why do we need this functions?

Reply:
Oh, yes, It is not necessary.  I'll take it out.